plugins/api-docs-module-wsdl-doc-backend/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@backstage/plugin-api-docs-module-wsdl-doc-backend",
+  "description": "Additional functionalities for the api-docs plugin that renders the output of WSDL",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "backend-plugin"
+  },
+  "homepage": "https://backstage.io",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/backstage",
+    "directory": "plugins/api-docs-module-wsdl-doc-backend"
+  },
+  "scripts": {
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack",
+    "clean": "backstage-cli package clean",
+    "start": "backstage-cli package start"
+  },
+  "dependencies": {
+    "@types/express": "^4.17.6",
+    "cross-fetch": "^3.1.5",
+    "express": "^4.17.1",
+    "express-promise-router": "^4.1.0",
+    "leasot": "^12.0.0",
+    "saxon-js": "^2.5.0",
```

Comment:
Sorry but the license of this dependency isn't one that I recognize. From what I can tell it might be a license that mixes in clauses from a couple of different existing ones. Unless we can reliably identity it as a license that is compatible with our [allowed](https://github.com/cncf/foundation/blob/main/allowed-third-party-license-policy.md#approved-licenses-for-allowlist) licenses I'm afraid we can't add it as a dependency in this repo.

One way around this is that you host this in a separate repo in order to separate it out. If you do I think it can perhaps be good to add a notice in the README too.